ElasticaAdmin/DisplayBlock/ElasticaSearchStrategy.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace OpenOrchestra\ElasticaAdmin\DisplayBlock;
+
+use OpenOrchestra\DisplayBundle\DisplayBlock\Strategies\AbstractStrategy;
+use OpenOrchestra\ModelInterface\Model\ReadBlockInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class ElasticaSearchStrategy
+ */
+class ElasticaSearchStrategy extends AbstractStrategy
+{
+    /**
+     * Check if the strategy support this block
+     *
+     * @param ReadBlockInterface $block
+     *
+     * @return boolean
+     */
+    public function support(ReadBlockInterface $block)
+    {
+        return 'elastica_search' === $block->getComponent();
```

Comment:
That was going to be one of my first refacto :).

To be sure of the constant position, I wanted to finish the Front part